src/code42cli/cmds/search/extraction.py

```diff
@@ -1,17 +1,20 @@
 import json
 
+import click
 from c42eventextractor import ExtractionHandlers
 from click import secho
 from py42.sdk.queries.query_filter import QueryFilterTimestampField
 
 import code42cli.errors as errors
 from code42cli.date_helper import verify_timestamp_order
 from code42cli.logger import get_main_cli_logger
+from code42cli.output_formats import get_dynamic_header
 from code42cli.util import warn_interrupt
 
 logger = get_main_cli_logger()
 
 _ALERT_DETAIL_BATCH_SIZE = 100
+_EVENT_COUNT_PER_PAGE = 10
```

Comment:
Need to decide a proper value or it can be a command line option, allowing user to define how many results to view.

Reply:
I was going to suggest 20, but I'm open to feedback on that. I think an option for specifying the number of lines to output is probably not necessary.

Reply:
It should be terminal height, which it does automatically!